scripts/calculate-hashes.ts

```diff
@@ -0,0 +1,122 @@
+import { ethers } from "ethers";
+import { readFileSync, writeFileSync } from "fs";
+import { hashBytecode } from "zksync-web3/build/src/utils";
+
+type Hashes = {
+  sourceCodeHash: string;
+  bytecodeHash: string;
+};
+
+type SystemContractsHashes = {
+  bootloader: Hashes;
+  defaultAA: Hashes;
+};
+
+const BOOTLOADER_SOURCE_CODE_PATH = "./bootloader/build/proved_batch.yul";
+const BOOTLOADER_BYTECODE_PATH =
+  "./bootloader/build/artifacts/proved_batch.yul/proved_batch.yul.zbin";
+const DEFAULT_ACCOUNT_SOURCE_CODE_PATH = "./contracts/DefaultAccount.sol";
+const DEFAULT_ACCOUNT_JSON_PATH =
+  "./artifacts-zk/cache-zk/solpp-generated-contracts/DefaultAccount.sol/DefaultAccount.json";
+const OUTPUT_FILE_PATH = "./SystemContractsHashes.json";
+
+const readFileAsHexString = (path: string, errorMessage: string): string => {
+  try {
+    return "0x" + readFileSync(path, "hex");
+  } catch {
+    throw new Error(errorMessage);
+  }
+};
+
+const loadBytecodeFromJson = (path: string, errorMessage: string): string => {
+  try {
+    const jsonFile = readFileSync(path, "utf8");
+    return JSON.parse(jsonFile).bytecode;
+  } catch {
+    throw new Error(errorMessage);
+  }
+};
+
+const getHashes = (
+  sourceCode: string,
+  bytecode: string,
+  errorMessage: string
+): Hashes => {
+  try {
+    return {
+      sourceCodeHash: ethers.utils.sha256(sourceCode),
+      bytecodeHash: ethers.utils.hexlify(hashBytecode(bytecode)),
+    };
+  } catch {
+    throw new Error(errorMessage);
+  }
+};
+
+const main = async () => {
+  const checkOnly = process.argv.includes("--check-only");
+
+  const bootloaderSourceCode = readFileAsHexString(
+    BOOTLOADER_SOURCE_CODE_PATH,
+    "Failed to read Bootloader source code. Make sure to run `yarn build-yul` before you run this script!"
```

Comment:
please also print  the CODE_PATH value in the error message (this usually makes it a lot easier to debug).

here and in other lines below

Reply:
path is printed